content/1.7/back-office/shop-parameters/customer-settings/view-titles.md

```diff
@@ -0,0 +1,68 @@
+---
+title: View titles
+weight: 1
+---
+
+### Call to action
+
+ - Add new social title, when pressed redirects to the [specs of the add/edit social titles]({{<ref "add-edit-social-titles.md">}} "Add/edit social titles") 
+
+### Settings wheel 
+
+1) Add new
+
+After clicking on it, redirects to the [specs of the add/edit social titles]({{<ref "add-edit-social-titles.md">}} "Add/edit social titles") 
+
+2) Refresh list
+
+After clicking on it, the page is reloaded. Sorts and searches are kept.
+
+3) Show SQL query
+
+After clicking on it, the SQL query is displayed in a modal. The SQL query takes into account filters and sorts that have been done.
+
+There are two buttons:
+
+- Close: After clicking on it, the modal is closed
+- Export to SQL Manager: After clicking on it, the SQL query is opened in Advanced parameters > Database > SQL Manager
+
+4) Export to SQL Manager
+
+After clicking on it, the SQL query is opened in Advanced parameters > Database > SQL Manager
+
+# Titles
+
+### Columns
+
+1) ID
+
+It's possible to sort by id asc or desc and search by id.
+
+2) Social title
+
+It's possible to sort by social title asc or desc and search by social title.
+
+3) Gender
+
+It's possible to sort by picking the gender from a drop-down menu.
+
+4) Image
+
+Shows an image of the tile.
+
+5) Actions
+
+**Edit.** The title can be edited as often as necessary.
+
+👉 See the [specs of the add/edit social title]({{<ref "add-edit-social-titles.md">}} "Add/edit social titles") 
+
+**Delete.** The title can be deleted by clicking on this button. After clicking on "Delete", a modal with the title "Delete selection" is opened asking to confirm or cancel the action: "Are you sure you want to delete the selected item(s)?"
+The action can be canceled by clicking on the cross or on the cancel button.
+After clicking on "Delete" button, the title is deleted, a successful message is displayed "Successful deletion."
+
+### Bulk actions 
+
+- Select all/Unselect all: this button will select/unselect all the checkboxes at the beginning of each row of the listing.
```

Comment:
This checkbox behavoir looks weird to me

Reply:
@fabienvallon the checkbox for "Select all" stays selected as an image, but I believe that might be intentional to separate the images from selected and non selected, but work as intended.

Not sure though if that is the correct way? @MatShir